classes/DevToolsCommand.php

```diff
@@ -308,11 +308,16 @@ protected function validate(string $type, $value)
     {
         switch ($type) {
             case 'name':
-                // Check If name
+                // Check if name is empty
                 if ($value === null || trim($value) === '') {
                     throw new \RuntimeException('Name cannot be empty');
                 }
 
+                // Check if name starts with a numeric character
+                if (is_numeric($value[0])) {
+                    throw new \RuntimeException('Name must start with a alphanumeric character [a-z|A-Z]');
```

Comment:
I assume it's meant to be just an alpha instead of alphanumeric :)

Reply:
Thanks for spotting! That was a typo.
An updates PR is in the making.